components/grove_tb6612fng.rst

```diff
@@ -14,11 +14,25 @@ At the current stage of implementation only DC motor is implemented.
 
 .. code-block:: yaml
 
+    #Enable I2C with scan to find address
+    i2c: 
+      sda: 4
+      scl: 5
+      scan: True
+
     # Example configuration grove motor
     grove_tb6612fng:
+      id: test_motor
+      address: 0x14
+    
+    # Example of multi config
+    grove_tb6612fng:
+      - address: 0x14
         id: test_motor
-        name: motor_outside
-        address: 0x14
+
+      - address: 0x15
+        id: test_motor2
+
```

Comment:
This whole example config should be smaller and only include the minimum required for getting the motor working. The rest is up to the user to read the documentation and write their config for their situation.